README.md

```diff
@@ -37,6 +37,19 @@ except: ['pipe'] // imports all helpers except `pipe`
 only: ['pipe'] // imports only `pipe`
 ```
 
+## Argument ordering
+
+Please note that the ordering of arguments is somewhat different then you might
```

Comment:
How about:

```md
This addon is built with _composability_ in mind, and in order to faciliate that,  
the ordering of arguments is somewhat different then you might be used to.

For all non-unary helpers, the subject of the helper function will always be the last argument.
This way the arguments are better readable if you compose together multiple helpers:

{{take 5 (sort "lastName" "firstName" (filter-by "active" array))}}

For action helpers, this will mean better currying semantics:

<button {{action (pipe (action "closePopover") (toggle "isExpanded") this)}}>
    {{if isExpanded "I am expanded" "I am not"}}
</button>
```